libs/designsystem/src/lib/components/toggle/toggle.component.scss

```diff
@@ -1,5 +1,12 @@
 @use '~@kirbydesign/core/src/scss/utils';
+@use '~@kirbydesign/core/src/scss/interaction-state';
 
 .toggle-checked {
   --background-checked: #{utils.get-color('success')};
 }
+
+ion-toggle {
+  @include interaction-state.apply-focus-part($part: 'track');
+  overflow: visible;
+  contain: none;
```

Comment:
I'm not completely comfortable with CSS Containment, yet. Maybe it's just me being ignorant, but I'd appreciate a short comment on why we need the `contain: none` declaration.

And just for all intents and purposes, did you verify that browser adaption of CSS Containment is ok?

Reply:
Added a comment to clarify this.